.gitmodules

```diff
@@ -1,3 +1 @@
-[submodule "papers/wg21"]
-	path = papers/P2988/wg21
```

Comment:
This paper is not related to iterator_interfaces (https://wg21.link/P2988. Left over from optional26 repo.